assignments/prototype-refactor.js

```diff
@@ -2,39 +2,40 @@
 // Today your goal is to refactor all of this code to use ES6 Classes.
 // The console.log() statements should still return what is expected of them.
 
-function GameObject(options) {
-  this.createdAt = options.createdAt;
-  this.dimensions = options.dimensions;
+class GameObject {
+  constructor(createdAt,dimensions) {
+  this.createdAt = createdAt;
+  this.dimensions = dimensions;
+}
+destroy() {
+   return `Object was removed from the game.`;
 }
-
-GameObject.prototype.destroy = function() {
-  return `Object was removed from the game.`;
-};
-
-function CharacterStats(characterStatsOptions) {
-  GameObject.call(this, characterStatsOptions);
-  this.hp = characterStatsOptions.hp;
-  this.name = characterStatsOptions.name;
 }
 
-CharacterStats.prototype = Object.create(GameObject.prototype);
 
-CharacterStats.prototype.takeDamage = function() {
+class CharacterStats extends GameObject {
+  constructor(hp,name) {
+  super('11/11/11', '1 ft by 2 ft');
```

Comment:
There seems to be a bit of confusion here by how `super` works. It is used to call the parent's constructor. 

Here is a more in-depth explanation: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/super